_posts/2024-06-25-diving-into-opensearch-2.15.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  Diving into OpenSearch 2.15
+authors:
+  - jamesmcintyre
+date: 2024-06-25
+categories:
+    - releases
+meta_keywords: opensearch 2.15, opensearch performance, hybrid search, opensearch parallel processing, opensearch batch processing, opensearch SIMD, opensearch vector performance, opensearch rolling upgrade, opensearch guardrails
+meta_description: OpenSearch 2.15.0 brings an array of new and expanded functionality designed to help users scale up performance and efficiency; advance stability, availability, and resiliency; and enhance your search, analytics, and observability applications, along with new machine learning capabilities and ease-of-use improvements.
+has_science_table: true
+featured_blog_post: false 
+featured_image: false # /assets/media/blog-images/__example__image__name.jpg
+---
+
+OpenSearch 2.15 is [now available](https://opensearch.org/downloads.html) with an array of new and expanded functionality to help you scale up performance and efficiency; advance stability, availability, and resiliency; and enhance your search applications, along with new machine learning (ML) capabilities and ease-of-use improvements. You can try out the latest updates using OpenSearch Dashboards on [OpenSearch Playground](https://playground.opensearch.org/app/home#/), and for a comprehensive rundown of what's new, check out the [release notes](https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.15.0.md). Here are some of the new tools you can start using with OpenSearch 2.15.
+
+### ***Cost, performance, scale***
+
+This release delivers a number of features focused on helping users improve **cost, performance, and scale** for their OpenSearch deployments.
+
+**Process documents faster with parallel ingestion**
```

Comment:
@krisfreedain: minor feedback for future blog posts: please use headings and not manual formatting here (e.g. use a `####` heading here).

reason: if you use a heading it'll get an `id` in the HTML code and you can set an anchor to that (e.g. `https://opensearch.org/path-to-blog#my-sub-heading-title`), making it easier to link to specific parts of the post

Reply:
thanks for the feedback @rursprung - I appreciate it!

@nateynateynate @jhmcintyre @pajuric - check this out.

Reply:
oh, and another good reason (as a colleague of mine never fails to point out): it's much better for people who use screen readers as it gives them context! it's a classic accessibility rule that you shouldn't manually format things where proper structuring elements (e.g. headings) exist